extensions-core/avro-extensions/src/main/java/org/apache/druid/data/input/avro/AvroFlattenerMaker.java

```diff
@@ -164,7 +166,7 @@ private Object transformValue(final Object field)
     } else if (field instanceof Utf8) {
       return field.toString();
     } else if (field instanceof List) {
-      return ((List<?>) field).stream().filter(Objects::nonNull).collect(Collectors.toList());
+      return ((List<?>) field).stream().filter(Objects::nonNull).map(this::transformValue).collect(Collectors.toList());
```

Comment:
this could possibly generate new output for a list which has `ByteBuffer` in it. But I think if anyone is using this then their data might look absurd since doing a `toString` on `ByteBuffer` doesn't return the backed bytes.